metrics-operator/controllers/common/providers/datadog/datadog.go

```diff
@@ -105,6 +168,17 @@ func (d *KeptnDataDogProvider) getSingleValue(points [][]*float64) float64 {
 	return sum / float64(count)
 }
 
+func (d *KeptnDataDogProvider) getResultSlice(points [][]*float64) []string {
+	resultSlice := make([]string, 0, len(points))
+	for _, point := range points {
+		if point[1] != nil {
```

Comment:
please also add a check for the length of `point` before accessing the element at index `1´ - otherwise we might crash with an index out of range error